skt/runner.py

```diff
@@ -485,6 +485,29 @@ def __jobsubmit(self, xml):
 
     def run(self, url, release, wait=False, host=None, uid="",
             arch=platform.machine(), reschedule=True):
+        """
+        Main method for testing in Beaker.
```

Comment:
Could you please use imperative mood, and descriptive instead of prescriptive language? Also, "method" would be redundant here. Something like "Run the tests" would work.